api/rest/property_value.md

```diff
@@ -6,156 +6,492 @@ parent: REST
-# Show Property Values for Node(s)
+# Retrieve property values
 
-Given a list of nodes and a property label, returns values associated with the
-given property for each node.
+This endpoint is suitable for situations in which you have a node or list of nodes and desire to obtain the values of specified properties attached to those nodes. 
+
+## General information about this endpoint
 
 **URL**: `/node/property-values`
 
-**Method**: `GET`, `POST`
+**Methods available**: `GET`, `POST`
 
-**Auth required**: Optional
+**Authentication**: Optional
 
-**Required Arguments**:
+**Required arguments**:
 
 *   `dcids`: A list of nodes to query, identified by their DCID.
-
 *   `property`: The property label to query for.
 
-**Optional Arguments**:
+**Optional arguments**:
 
-*   `valueType`: The type of the property value to filter by, only applicable if
+*   `valueType`: The type of the property value to filter by. Only applicable if
     the value refers to a node.
-
-*   `direction`: The direction of the label, valid values are "out" and "in".
-
+*   `direction`: The label's direction. Only valued as `out` (returning response nodes directed towards the requested node) or `in` (returning response nodes directed away from the request node).
 *   `limit`: (≤ 500) Maximum number of values returned per node.
-
 *   `key`: Your API key.
 
-## GET Request
-
-**Example**
+## How to construct a request to the property value endpoint
 
-*   Get name of two states
+### Step 1: assembling the information you will need
 
-    ```bash
-    curl 'https://api.datacommons.org/node/property-values?dcids=geoId/05&dcids=geoId/06&property=name'
-    ```
+This endpoint requires two arguments and offers four additional optional arguments, as listed:
 
-*   Query with type filter and limit constraint
+ - `dcids`: Data Commons uniquely identifies nodes by assigning them DCIDs, or Data Commons IDs. Your query will need to specify the DCIDs for the nodes of interest.
+ - `property`: The property whose value you are interested in.
 
-    ```bash
-    curl 'https://api.datacommons.org/node/property-values?dcids=geoId/05&property=location&valueType=Election&limit=5'
-    ```
+In addition to these required properties, this endpoint also allows four optional arguments.
 
-*   Query with direction
+  - `valueType`: If the property queried only takes on node values, you can use this argument to filter nodes in the response, ensuring the response only contains nodes with the specified type.
 
-    ```bash
-    curl 'https://api.datacommons.org/node/property-values?dcids=geoId/05&property=containedInPlace&direction=out'
-    ```
+  - `direction`: You can specify this argument as `out` to indicate that you desire the response to only include nodes which are supercategories of the specified `DCIDs`, or `in` to only return nodes that are subcategories of the specified `DCIDs`. (For example, South America is a supercategory of Argentina, which in turn is a supercategory of Buenos Aires.)
```

Comment:
I think this is a good example. I wonder if it would be even clearer with a small node image, with pointed arrows to indicate direction, and some example queries (i.e. how does a user translate that example into property value queries?)

Reply:
I added an image next to Example 5, the query that determines the country in which Buenos Aires is located. Does that location seem good to you? Furthermore, would you like for me to also add queries for the other three arrows on the diagram (ie, the continent on which Argentina is located, countries on the continent of South America, and also cities/administrative areas within Argentina)?